apps/explorer/src/app/routes/txs/home/txs-home-block-height-wrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import { useFetch } from '@vegaprotocol/react-helpers';
+import { Txs } from './index';
+import { DATA_SOURCES } from '../../../config';
+import type { TendermintBlockchainResponse } from '../../blocks/tendermint-blockchain-response';
+
+export const TxsHomeBlockHeightWrapper = () => {
+  const {
+    state: { data, error, loading },
+    refetch,
+  } = useFetch<TendermintBlockchainResponse>(
+    `${DATA_SOURCES.tendermintUrl}/blockchain`
+  );
+
+  const latestBlock = data?.result?.block_metas?.[0]?.header?.height;
+
+  if (error) {
+    return <div>{`Something went wrong: ${error.message}`}</div>;
+  }
+
+  if (loading) {
+    return <div>{'Getting latest block height...'}</div>;
+  }
```

Comment:
chore (blocking): is there a reason we are not using AsyncRenderer? Seems like duplciated logic from there. I know we removed this from the child of this component, but seems like it would be useful here

Reply:
No probs - it was a preference for improved messaging, but I'll refactor AsyncRenderer to support custom messaging :)